RLTest/redis_std.py

```diff
@@ -349,12 +351,29 @@ def _stopProcess(self, role):
                         p.wait()
                     except:
                         pass
-            process.terminate()
-            while True:
-                if process.poll() is None:  # None returns if the processes is not finished yet, retry until redis exits
-                    time.sleep(0.1)
-                else:
-                    break
+
+            if self.terminateRetries is None:
+                # ask once, then wait for process to exit
```

Comment:
Probably should have a maximum period after which this breaks

Reply:
This is currently handled by CirlceCi `no_output_timeout`

Reply:
This is the existing code, in which we wait infinitely. Last time we touched it, Avi removed the extra calls to `process.terminate()` because it created a problem to the sanitizer. So I wanted to be conservative here, and trigger repetitive stops only upon explicit request.

Reply:
@oshadmi I'd like to avoid logic in CircleCI, so we need to solve everything in RLTest/module level. I say, let's keep the old code as-is, and try improving on it in the RedisGraph context.